fickling/hook.py

```diff
@@ -0,0 +1,128 @@
+import pickle
+
+import torch
+
+import fickling.loader as loader
+import fickling.polyglot as polyglot
+from fickling.fickle import Pickled
+from fickling.pytorch import PyTorchModelWrapper
+
+
+def hook_pickle_load(pickle_load_function):
+    # Wraps the function to actually run the core loader
+    def wrapper(*args, **kwargs):
+        file = args[0]
+        return loader.load(file)
```

Comment:
This should also pass `*args` and `**kwargs`, see my comment on `fickling/loader.py`